src/Algolia.Search/Clients/AnalyticsConfig.cs

```diff
@@ -30,7 +30,7 @@ namespace Algolia.Search.Clients
     /// <summary>
     /// Analytics client configuration
     /// </summary>
-    public class AnalyticsConfig : AlgoliaConfig
+    public sealed class AnalyticsConfig : AlgoliaConfig
```

Comment:
This is not a breaking change for people that have extended already the `AnalyticsConfig`?

Reply:
Yes it is, but there is not point of extending it since the added values are not going to be used by the clients.